addons/website/tests/test_ui.py

```diff
@@ -216,3 +216,7 @@ def test_11_website_snippet_background_edition(self):
             'mimetype': 'image/png',
         })
         self.start_tour('/', 'snippet_background_edition', login='admin')
+
+    def test_12_edit_translated_page_redirect(self):
+        self.start_tour("/nl_BE", 'edit_translated_page_redirect', login='admin', timeout=120)
```

Comment:
/nl_BE/ will be a 404 no ? 
where do you install this one ?

+ please add task-id in commit task-2622270

Reply:
Yes, the language is still missing. I still have to figure out how to add it in the test. I'll fix the other remarks as well.

Reply:
You can check the `test_lang_url` test file